src/drivers/clock/e2k/e2k.c

```diff
@@ -94,12 +96,17 @@ static int e2k_clock_config(struct time_dev_conf * conf) {
 }
 
 static cycle_t e2k_clock_read(void) {
+#if 0
+	/* TODO Should be checked, because values are looking a bit weird. */
 	return e2k_read32((void*)E2K_RESET_COUNTER_LOW); /* Ignore high 32 bits */
+#endif
+	return 0;
 }
 
 static struct time_event_device e2k_clock_event = {
 	.config   = e2k_clock_config,
 	.event_hz = 1000,
```

Comment:
May be use `option number freq` instead?

Reply:
Fixed.